src/open-api.ts

```diff
@@ -33,13 +36,30 @@ interface GeneratorParams {
   errorResponseDescription?: string;
   /** @default true */
   hasSummaryFromDescription?: boolean;
+  /**
+   * @desc Used for comparing schemas wrapped into z.lazy() to limit the recursion
+   * @default JSON.stringify()
```

Comment:
actually hashing is also a part of serialization involved in comparison.
should extract it and adjust the docs

Reply:
fixed